.circleci/config.yml

```diff
@@ -226,6 +229,46 @@ jobs:
           name: Publish package
           working_directory: packages/vue-component-library
           command: npm publish --registry https://$NPM_REGISTRY
+  build_docs-site:
+    executor: node
+    working_directory: ~/standards-toolkit
+    steps:
+      - checkout
+      - dependencies
+      - attach_workspace:
+          at: ~/standards-toolkit
+      - run:
+          name: Build site
+          working_directory: packages/docs-site
+          command: npm run build
+      - persist_to_workspace:
+          root: .
+          paths:
```

Comment:
Will this approach also cause all the node_modules to get persisted?

Reply:
It will only persist the paths specified below:

```
- packages/docs-site/public
- packages/docs-site/firebase.json
- packages/docs-site/.firebaserc
```

I'm only carrying over the build artefacts for the two deployment jobs.

I guess you could use it to persist `node_modules` if you wanted but there is cacheing for that?

Reply:
It's my understand that it only persists the paths specified:

https://circleci.com/docs/2.0/configuration-reference/#example-for-root-key

Reply:
re: workspace root question

When learning about workspaces, I used an official CircleCI example of workspace-forwarding and referenced that for best practice:

https://github.com/CircleCI-Public/circleci-demo-workflows/blob/workspace-forwarding/.circleci/config.yml